.cicd/helpers/docker-hash.sh

```diff
@@ -16,5 +15,11 @@ function determine-hash()
     export HASHED_IMAGE_TAG="${IMAGE_TAG}-${DETERMINED_HASH}"
 }
 
-determine-hash ".cicd/${IMAGE_TAG}.dockerfile"
-export FULL_TAG="eosio/producer:eos-vm-$HASHED_IMAGE_TAG"
```

Comment:
Let's go back to this. I don't think we need the added complexity of error messages, since these scripts are not meant for the end-user. If a developer can't readily understand what they are doing and why their command without an image tag doesn't work, then I think we have failed and these scripts are too complicated.

Reply:
What if a blockchain guy removes it from the pipeline.yml? They'd want to get some sort of feedback in the failure logs.